modules/ROOT/pages/6.0-release-notes-plugins.adoc

```diff
@@ -0,0 +1,300 @@
+
+[[table]]
+==== Table
+
+The `table` plugin’s functionality is, as of {productname} 6.0, entirely focussed on the user interface (UI) for creating and editing tables.
+
+The code that generates tables is now a Core part of {productname} 6.0.
+
+This, now core feature of {productname} 6.0, also includes the following fixes, changes, and improvements.
+
+* If the selected cells in the `tfoot` section of a table were header elements (ie were `th` elements), pressing the `table` row header toolbar button (which invokes the `mceTableRowType` command) returned the row as a header row.
+
+This was incorrect: the section takes precedence and cells in the `tfoot` section of a table should be declared as a footer row.
+
+In {productname} 6.0, selecting cells in the `tfoot` section of a table and pressing the row header toolbar button returns the row as a footer row, as expected.
+
+* `mceInsertTableDialog` is the new command for invoking an _Insert Table_ dialog box.
++
+The previous command, `mceInsertTable`, no longer works to invoke an _Insert Table_ dialog box: use the `mceInsertTableDialog` command instead.
++
+The `mceInsertTable` command is still the command for inserting a table with specified properties directly and without asking for user-input.
++
+* The default values for the `table_style_by_css` and the `table_use_colgroups` options have both been changed.
++
+Previously, these options defaulted to `false`. As of {productname} 6.0, both options now default to `true`.
++
+This is in line with modern web practice.
++
+* the `table_responsive_width` option was previously deprecated and superseded by `table_sizing_mode`.
++
+In {productname} 6.0, the `table_responsive_width` option has been removed.
+
+//[[fixed-plugins]]
+//=== Fixed plugins
+//
+//[id-added-plugins]]
+//=== Added plugins
+
+[[removed-or-deprecated-plugins]]
+=== Removed or deprecated plugins
+
+[[bbcode]]
+==== BBCode
+
+As per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the `bbcode` plugin has been removed. It is no longer part of {productname} 6.0. And it is not otherwise available.
+
+[[color-picker]]
+==== Color Picker
+
+Color Picker’s functionality has been a built in part of {productname} since version 5.
+
+Consequently, and as per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the empty `colorpicker` plugin has been removed.
+
+[[context-menu]]
+==== Context Menu
+
+Context Menu’s functionality has been a built in part of {productname} since version 5.
+
+Consequently, and as per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the empty `contextmenu` plugin has been removed.
+
+[[full-page]]
+==== Full Page
+
+As per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the `fullpage` plugin has been removed. It is no longer part of {productname} 6.0. And it is not otherwise available.
```

Comment:
```suggestion
As per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[{productname} 5 announcement, window="_blank"], the `fullpage` plugin has been removed. It is no longer part of {productname} 6.0. And it is not otherwise available.
```